pkg/resolver/testHelper_test.go

```diff
@@ -96,18 +96,13 @@ func (r *Row) Scan(dest ...interface{}) error {
 	return nil
 }
 
-// ====================================================
-// Mock the Rows interface defined in the pgx library.
-// https://github.com/jackc/pgx/blob/master/rows.go#L24
-// ====================================================
-
 //Prop will be the property input for searchComplete
 func newMockRowsWithoutRBAC(mockDataFile string, input *model.SearchInput, prop string, limit int) *MockRows {
 	// Read json file and build mock data
 	bytes, _ := ioutil.ReadFile(mockDataFile)
 	var data map[string]interface{}
 	if err := json.Unmarshal(bytes, &data); err != nil {
-		panic(err)
+		klog.Warning("Error unmarshaling data", err.Error())
```

Comment:
We avoid using panic() in the code because we don't want to kill the pod, but in the tests is ok because this will cause the test to fail later, so it's easier to debug if we panic here.

Reply:
Okay, that makes sense. Will change back to panic. thanks for explaining.